packages/material-ui/src/Radio/Radio.js

```diff
@@ -148,8 +148,8 @@ Radio.propTypes = {
    * Callback fired when the state is changed.
    *
    * @param {object} event The event source of the callback.
-   * You can pull out the new value by accessing `event.target.value`.
-   * @param {boolean} checked The `checked` value of the switch
+   * You can pull out the new value by accessing `event.target.value` (string).
```

Comment:
⬆️

Reply:
This should be updated too with the same info as in the previous commit